storybook/stories/asset/dummyData/data.js

```diff
@@ -1,22 +1,32 @@
 export const tinderCardDummyData = [
+  {
+    id: '0',
+    title: 'Blueberry',
+    content: 'perennial flowering plants with blue– or purple–colored berries.',
+    image: { uri: 'https://cdn.pixabay.com/photo/2016/04/13/07/18/blueberry-1326154_1280.jpg' },
+  },
   {
     id: '1',
-    image: require('../images/dummy_image_1.jpg'),
+    title: 'Strawberry',
+    content: 'a widely grown hybrid species of the genus Fragaria, collectively known as the strawberries, which are cultivated worldwide for their fruit',
```

Comment:
Test about `TinderCard` is failed.
11:1  error  This line has a length of 155. Maximum allowed is 120  max-len
29:1  error  This line has a length of 189. Maximum allowed is 120  max-len